airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -30,11 +31,13 @@
 from airflow.api_fastapi.common.db.common import get_session
 from airflow.api_fastapi.common.router import AirflowRouter
 from airflow.api_fastapi.execution_api.datamodels.taskinstance import (
+    TIDeferredStatePayload,
     TIEnterRunningPayload,
     TIHeartbeatInfo,
     TIStateUpdate,
     TITerminalStatePayload,
 )
+from airflow.models import Trigger
```

Comment:
```suggestion
from airflow.models.trigger import Trigger
```

for consistency with L41

Reply:
Handled